src/utils/spørsmålogsvar.ts

```diff
@@ -11,7 +11,7 @@ import { IntlShape } from 'react-intl';
 export const returnerJaNeiSvar = (
   spørsmål: IJaNeiSpørsmål,
   svar: IJaNeiSvar,
-  søknadobj: ISøknad
+  søknadobj: any
```

Comment:
Hvorfor any?

Reply:
Mistake! Fikset :) Var for å teste noe tidligere.